conda-store-server/tests/user_journeys/helpers/helpers.py

```diff
@@ -0,0 +1,145 @@
+"""Helper functions for user journeys."""
+from datetime import datetime, timedelta, timezone
```

Comment:
I renamed to utils, but kept the nested structure so that I could split `api_utils` from `time_utils`, I know that adding the `_utils` suffix is a little awkward, but it stops name collisions.